receiver/elasticsearchreceiver/metadata.yaml

```diff
@@ -107,15 +142,42 @@ metrics:
       value_type: int
     attributes: [cache_name]
     enabled: true
-  elasticsearch.node.fs.disk.available:
-    description: The amount of disk space available across all file stores for this node.
+  elasticsearch.node.disk.free:
+    description: Amount of free disk space on the node.
     unit: By
     sum:
       monotonic: false
       aggregation: cumulative
       value_type: int
     attributes: []
     enabled: true
+  elasticsearch.node.disk.available:
+    description: Amount of available space on the Java virtual machine. This is the actual amount of free disk space the Elasticsearch node can utilise.
```

Comment:
The descriptions on these two metrics are helpful in differentiating them, but the names seem too closely related. Should one of these be in an alternate namespace? Perhaps the JVM one should be in one of the existing `jvm.*` namespaces?

Reply:
The git diff is misleading here but `elasticsearch.node.disk.available` used to be `elasticsearch.node.fs.disk.available` prior to my changes. `elasticsearch.node.fs.disk.available` is a [current metric](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/elasticsearchreceiver/metadata.yaml#L110)  so changing it would be backwards breaking.

Given how `elasticsearch.node.disk.free` isn't as useful, I will:
- Remove `elasticsearch.node.disk.free`
- Revert back to `elasticsearch.node.fs.disk.available`.